reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -324,7 +322,7 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         result: ScanResult,
         findings: MutableList<EvaluatedFinding>,
         pkg: EvaluatedPackage,
-        licenseFindingCurations: Collection<LicenseFindingCuration>
+        id: Identifier
```

Comment:
This is the same as `pkg.id`, so you can drop the parameter instead.

Reply:
Done.